rigex-front-end/src/store/signUp/reducers.js

```diff
@@ -1,8 +1,8 @@
 import actions from './actionTypes';
 
-const loginReducer = (state = {}, action) => {
+const signUpReducer = (state = {}, action) => {
```

Comment:
same thing here, this is the loginReducer in store/login/reducers. Should probably be in store/register/reducers